kitsune/customercare/forms.py

```diff
@@ -10,21 +11,34 @@ class ZendeskForm(forms.Form):
     """Form for submitting a ticket to Zendesk."""
 
     product = forms.CharField(disabled=True, widget=forms.HiddenInput)
-    category = forms.ChoiceField(
-        label=_lazy("What do you need help with?"), choices=CATEGORY_CHOICES
+    email = forms.EmailField(
+        label=_lazy("Email"), required=False, disabled=True, widget=forms.HiddenInput
     )
+    category = forms.ChoiceField(label=_lazy("What do you need help with?"))
     os = forms.ChoiceField(
         label=_lazy("What operating system does your device use?"),
         choices=OS_CHOICES,
         required=False,
     )
     subject = forms.CharField(label=_lazy("Subject"), required=False)
-    description = forms.CharField(label=_lazy("Your message"), widget=forms.Textarea())
+    description = forms.CharField(label=_lazy("Tell us more"), widget=forms.Textarea())
     country = forms.CharField(widget=forms.HiddenInput)
 
-    def __init__(self, *args, product, **kwargs):
+    def __init__(self, *args, product, user=None, categories=None, **kwargs):
         kwargs.update({"initial": {"product": product.slug}})
         super().__init__(*args, **kwargs)
+        self.fields["category"].choices = categories if categories is not None else []
```

Comment:
Why do we need this?

Reply:
We have to set the categories on the displayed form on the fly based on which flavor form we are showing - logged in or not logged in.  The `self.fields["category"]` piece has to come after the `super` in order to alter the field.

Reply:
After a hard look at your prior comment, I'm changing this all out.